docs/operations/images.md

```diff
@@ -268,7 +268,9 @@ az vm image list --all --output table \
 
 ### Ubuntu 24.04 (Noble)
 
-Support for Ubuntu 24.04 is **experimental**.
+Support for Ubuntu 24.04 is based on Kernel version **6.8**.
```

Comment:
```suggestion
Support for Ubuntu 24.04 is based on Kernel version **6.8**.

**WARNING**: The Amazon VPC CNI is not compatible with Ubuntu 24.04.
```

Reply:
is there a github issue we can link to? I'm not finding one

Reply:
Not that I know of, I just remember it now working.